spec/abilities/embargo_and_lease_ability_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'cancan/matchers'
+
+describe "Ability on embargos and leases" do
+  subject { Ability.new(current_user) }
+
+  let(:current_user) { create(:user) }
+
+  describe "a regular user" do
+    it do
```

Comment:
http://i1.kym-cdn.com/entries/icons/original/000/012/059/they-dont-think-it-be-like-it-is-but-it-do.jpg